plugin.video.skyvideoitalia/resources/lib/skyitalia.py

```diff
@@ -0,0 +1,220 @@
+# -*- coding: utf-8 -*-
+import datetime
+import html
+import json
+import re
+import socket
+import urllib.request as urllib2
+
+from . import addonutils
+from simplecache import SimpleCache
+
+
+class SkyItalia:
+    HOME = 'https://video.sky.it/'
+    GET_VIDEO_SEARCH = 'https://video.sky.it/be/getVideoDataSearch?token={token}&section={section}&subsection={subsection}&page={page}&count=30'  # noqa: E501
+    GET_PLAYLISTS = 'https://video.sky.it/be/getPlaylistInfo?token={token}&section={section}&subsection={subsection}&start=0&limit=31'  # noqa: E501
+    GET_PLAYLIST_VIDEO = 'https://video.sky.it/be/getPlaylistVideoData?token={token}&id={id}'  # noqa: E501
+    GET_VIDEO_DATA = 'https://apid.sky.it/vdp/v1/getVideoData?token={token}&caller=sky&rendition=web&id={id}'  # noqa: E501
+    TOKEN = 'F96WlOd8yoFmLQgiqv6fNQRvHZcsWk5jDaYnDvhbiJk'
+    TIMEOUT = 15
+    LOGLEVEL = int(addonutils.getSetting('LogLevel'))
+    QUALITY = addonutils.getSetting('Quality')
+    QUALITIES = ['web_low_url', 'web_med_url', 'web_high_url', 'web_hd_url']
+    LOGOSDIR = '%sresources\\logos\\' % addonutils.PATH_T
+    FANART = addonutils.FANART
+
+    def __init__(self):
+        socket.setdefaulttimeout(self.TIMEOUT)
```

Comment:
Why do you set the socket here? It's not used anywhere? And you already set the timeout at the urlopen level?

Reply:
you're right. It's leftover code before I put the timeout in the urlopen... I'll fix it.

Reply:
Sure, give me a ping if it's done.